config.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget android-versionCode="6" id="uk.ac.ucl.homeapp" ios-CFBundleIdentifier="uk.ac.ucl.homeapp" ios-CFBundleVersion="6" version="0.7.6" xmlns:android="http://schemas.android.com/apk/res/android">
+<widget android-versionCode="8" id="uk.ac.ucl.homeapp" ios-CFBundleIdentifier="uk.ac.ucl.homeapp" ios-CFBundleVersion="8" version="0.7.8" xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Bundle version 8 already exists for iOS due to the bug we had on ios during last release. Not sure if this would work.